src/resources/views/fields/video.blade.php

```diff
@@ -172,7 +172,7 @@
 
                 var fetchVimeo = function( videoId, callback ){
 
-                    var api = 'http://vimeo.com/api/v2/video/' + videoId + '.json?callback=?';
+                    var api = 'https://vimeo.com/api/v2/video/' + videoId + '.json?callback=?';
```

Comment:
Doesn't this just switch the problem over (i.e. now a HTTP site will complain)?

Surely it would be better to check whether the main site is being served via HTTP or HTTPS?